packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gadgetinc/react",
-  "version": "0.4.1",
+  "version": "0.4.2",
```

Comment:
Oh, one thing I forgot: I'm not sure how we're publishing now, but I think it would be good for this change to be the last commit.

Reply:
Assuming the tests pass, our [release action](https://github.com/gadget-inc/js-clients/blob/main/.github/workflows/release.yml) will pick up the change and immediately publish the new version.

Reply:
Then probably good to have the bump at the end. I think that better reflects reality. It would feel weird to have the bump in one commit, and changes after it also included in that release, eh?

Reply:
Oh, good question. I know @scott-rc did some work on that recently: Scott, do you know if that commit order helps/matter at this point?

Reply:
From the release action's perspective, it won't matter. However, from our git history perspective, the version was changed on the first commit but the published package will have 3 additional commits inside of it.